ansible/playbooks/roles/redpanda_broker/templates/configs/defaults.j2

```diff
@@ -10,23 +10,23 @@ node:
     - address: {{ hostvars[inventory_hostname].advertised_ip }}
       port: {{ redpanda_kafka_port }}
     advertised_rpc_api:
-      address: {{ hostvars[inventory_hostname].advertised_ip }}
+      address: {{ hostvars[inventory_hostname].private_ip }}
```

Comment:
this must always be private or the cluster cannot talk to itself correctly

Reply:
This makes sense